### PR TITLE
Feat/export import style object

### PR DESCRIPTION
**Pull request by LuiseBrandenburger**
quick fix to use existing styleId on uploaded GeoJson

### CODE REVIEW

**Comment by LuiseBrandenburger** [comment]
I added a quick fix so "deleted" layers are not downloaded anymore. the Masterportal function to delete layers is actually not deleting them, it just sets the showInLayerTree option to false.

**Comment by ZenVega** [comment]
The PR doesn't include the style import yet, right?

File: addons/addons_3_0_0/projectDownloader/components/ProjectDownloader.vue [hunk 1]
```diff
@@ -44,9 +44,26 @@ export default {
...
+          elements: layerConfig.subjectlayer.elements.filter(
+            (layer) => layer.showInLayerTree === true,
+          ),
+        };
+
+        const filteredLayerConfig = {
+          baselayer: filteredBaselayer,
+          subjectlayer: filteredSubjectlayer,
+        };
+
```
Comment (ZenVega) [inline addons/addons_3_0_0/projectDownloader/components/ProjectDownloader.vue:63]: Nice!

File: addons/addons_3_0_0/projectUploader/store/actionsProjectUploader.js [hunk 2]
```diff
@@ -573,149 +573,151 @@ export default {
...
-            defaultColor,
-            defaultColor,
-            geometryType,
-            defaultPointSize,
-            1,
-            1,
-          );
+    if (datasrc.layer.values_.styleId === "default") {
+      vectorLayer.setStyle((feature) => {
+        const drawState = feature.getProperties().drawState;
```
Comment (ZenVega) [inline addons/addons_3_0_0/projectUploader/store/actionsProjectUploader.js:578]: When does a previously imported layer have a drawState?

**Review by ZenVega — APPROVED** [review]
Looking good! Merge ahead

**Pull request merged**